content/release-10-14-0/applications-10-14-0-bundle/10-14-0-0.md

```diff
@@ -1,12 +1,12 @@
----
-weight: 40
-title: Release 10.14.0.28
-layout: bundle
----
-
-<!--10.13.1.0 - 10.13.168.0; 14.0.0.1 - 14.0.0.28 -->
-
-### Improvements
```

Comment:
The change in this file looks weird. Did you change anything general in the layout or why does it show everything as deleted and added?

Reply:
No, I didn't change the layout. I don't know why it shows everything as deleted and added.